### ISSUE TITLE
HDR and monochrome images

### ISSUE THREAD

**Issue by 0xC0000054** [AOMediaCodec/av1-avif#203]
Is it valid for a monochrome image to use the HDR transfer curves?

ITU-R BT.2100-2 does not appear to have any references to monochrome images, but the transfer curve should work the same way for both monochrome and RGB data.
SMPTE 428-1-2006 also makes not mention of monochrome images, and I ma not sure if its transfer function is implied to require RGB.

I created two grayscale test images in Photoshop that have the PQ or SMPTE 428 transfer curves applied when saving.
The images appear to load fine in Krita, but there are some tone differences in Chrome 107.0.5304.107.

It appears that Krita only applies the PQ and SMPTE 428 transfer curves to RGB images, grayscale images are saved as SDR.

[hdr monochrome.zip](https://github.com/AOMediaCodec/av1-avif/files/10048180/hdr.monochrome.zip)

**Comment by leo-barnes** [comment]
> Is it valid for a monochrome image to use the HDR transfer curves?

Yes, that should be valid AFAIK. Monochrome images can be tagged with transfer curves, otherwise you would have no way of knowing if they are in linear or gamma space.
Unless the specs that describe the curves somehow state that they don't work for monochrome images I can't see why HDR curves should not apply.

**Comment by 0xC0000054** [comment]
> Unless the specs that describe the curves somehow state that they don't work for monochrome images I can't see why HDR curves should not apply.

The math for SMPTE 428-1-2006 and ITU-R BT.2100-2 HLG appears to assume RGB data.
But it appears that ITU-R BT.2100-2 PQ may support monochrome:

"F<sub>D</sub> is the luminance of a displayed linear component {R<sub>D</sub>, G<sub>D</sub>, B<sub>D</sub>} or Y<sub>D</sub> or I<sub>D</sub>, in cd/m2."

I am currently thinking I will disable HDR saving for monochrome images, this is probably the safest option in case I have misunderstood the various specs.
